Proofreading of the General section (#12)
simpleanalytics/docs: _docs/10_general/01_introduction.md

permalink: /
---

> Welcome to the Simple Analytics documentation! Here you will find detailed information on our features and how to customize them.

Reviewer: I like the [Stripe way](https://stripe.com/docs) of welcoming: "Whether you’re a startup or a global enterprise, learn how to integrate with Stripe to accept payments and manage your business online."

```suggestion
Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.
```

Something like this?

Author: I like this too 👍

> Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.

Reviewer: Should it be "needed consent" or "needing consent"?

Reviewer: Let me know what this should be @jakejarvis so I can merge your PR.

Author: Hmm, probably "needing consent" in that case but how does something like this sound split into two sentences?

"Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics for privacy friendly analytics. No consent required from your visitors!"

> Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics to move to privacy friendly analytics for your visitors without needed consent.

Reviewer: ```suggestion
Whether you’re a startup or a global enterprise, learn how to integrate with Simple Analytics for privacy friendly analytics. No consent required from your visitors!
```
